### PR TITLE
Text change on portfolio to be more towards programming experience etc

### PR DESCRIPTION
**Pull request by petros-chantz**
Magkiori koita ligo thn allagh tou text kai pesmou ama eina kalo. Hthela na to kanw na fainetai ligo pio towards programming experience. Plus pes mou ama uparcei kanena lathows

### CODE REVIEW

File: src/pages/Experience.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default function Experience() {
           academic research projects, in design research labs and in commercial
           projects for companies such as Signify (Ex. Philips Lighting) has
           helped me shape my working attitude and gain new insights on improving
-          myself.
+          my expertises.
```
Comment (rania-chantz) [inline src/pages/Experience.js:50]: den 3erw an to expertise exei plh8untiko, nomizw pws oxi

Comment (rania-chantz) [inline src/pages/Experience.js:50]:
```suggestion
          my expertise.
```

**Comment by rania-chantz** [comment]
des ama 8es na kaneis commit to suggestion

**Comment by petros-chantz** [comment]
kale pes ligo kalutero einai to text twra ?

**Comment by petros-chantz** [comment]
okay to kanw merge egw

**Pull request merged**